wewatch-client/package.json

```diff
@@ -17,7 +17,8 @@
     "react-scripts": "3.4.4",
     "reactstrap": "^8.7.1",
     "socket.io": "^2.3.0",
-    "styled-components": "^5.2.0"
+    "styled-components": "^5.2.0",
+    "underscore": "^1.12.0"
```

Comment:
Why did we need to the underscore library here? Was there a hidden bug we didn't catch before?

Reply:
Yeah! I believe that is why I was running into a compile error earlier. I added yarn underscore which seemed to resolve the issue.